.github/workflows/pylint.yml

```diff
@@ -0,0 +1,23 @@
+name: Pylint
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10"]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+         pip install -r dev-requirements.txt
+    - name: Analysing the code with pylint
+      run: |
+        pylint $(git ls-files '*.py')
```

Comment:
Looks good, but can you join this with the existing GH Action (the one that executes the tests)? You just need to copy out this single task 

```
    - name: Analysing the code with pylint
      run: |
        pylint $(git ls-files '*.py')
```

Reply:
I merged the develop branch into this feature branch. As far as I understand that is the rebase?
The two github actions have been compressed into one file (action) and I added the line where I deleted too much.